css/styles.css

```diff
@@ -0,0 +1,203 @@
+
+
+
+
+/* header y menu */
+.menu{
+    display:flex;
+    justify-content: space-between;
+    align-items:center;
+    padding:2%;
+}
+
+.menu ul{
+    display:flex;
+    list-style: none;
+    margin-right:15%;
+    
+}
+
+.menu ul a{
+    margin:10px;
+    text-decoration:none;
+    color:gray;
+}
+
+.menu h1{
+    text-transform: uppercase;
+    font-weight: 700;
+}
+.menu ul a:first-child{
+    color:#ed564b;
+}
+.menu ul a:hover{
+    color:#ed564b
+}
+
+/* main */
+
+
+
+
+.slider{
+    display:flex;
+    flex-direction: column;
+    align-items:center;
+    background-image: url(/home/diego/OneDrive/Ironhack/week1/day2/lab-bootstrap-cloning-revera/img/slider-4.jpg);
```

Comment:
Si hacéis uso de rutas absolutas no serán válidas en otro equipo diferente, debéis hacer uso de rutas relativas en este caso.